.github/workflows/pending-review-notification.yml

```diff
@@ -1,8 +1,8 @@
 name: Send pending review notifications to reviewer on github-discussion
 on:
   schedule:
-    # Every Tuesday and Thursday at 00:00 UTC.
-    - cron: '0 0 * * 2,5'
+    # Every Tuesday and Thursday at 16:00 UTC.
```

Comment:
Approving!! But PTAL at this. 
Shouldn't it be Tuesday and Friday according to this [link](https://crontab.guru/#0_16_*_*_2,5)

Reply:
@SAEb-ai Thanks for raising it, unexpectedly the old changes had incorrect code comment, I've updated it now!